src/services/pdf-styles.js

```diff
@@ -307,10 +307,10 @@ const docDefinitionGenerator = (content, pdfData) => {
 
 const fontDescriptors = {
   Roboto: {
```

Comment:
Presumably this should no longer be called "Roboto" but you need to check if changing would have impact

Reply:
When changing the name from Roboto to Arial it wasnt happy - so i kept it as roboto